component-catalog-connectors/airflow-example-components-connector/README.md

```diff
@@ -0,0 +1,60 @@
+## Elyra pipeline component examples catalog
+
+This catalog connector provides access to example pipeline components for Apache Airflow.
+
+### Install the component examples
+
+You can install the component examples from PyPI or source code. Note that a **rebuild of JupyterLab is not required**.
+
+**Prerequisites**
+
+- [Install Elyra](https://elyra.readthedocs.io/en/stable/getting_started/installation.html) (version 3.3 and above).
+
+**Install from PyPI**
+
+  ```
+  $ pip install elyra-airflow-example-components-catalog
+  ```
+
+**Install from source code**
+
+   ```
+   $ git clone https://github.com/elyra-ai/examples.git
+   $ cd examples/component-catalog-connectors/airflow-example-components-connector/
+   $ make clean source-install
+   ```
+
+### Use the connector
+
+1. Launch JupyterLab.
+1. [Open the '`Manage Components`' panel](
+https://elyra.readthedocs.io/en/stable/user_guide/pipeline-components.html#managing-custom-components-using-the-jupyterlab-ui).
+1. Add a new component examples catalog ('`+`' > '`New Apache Airflow example components catalog`').
+1. Specify a catalog name, e.g. '`Elyra example components for Kubeflow`'.
```

Comment:
Did you mean to reference Kubeflow here?
```suggestion
1. Specify a catalog name, e.g. '`Elyra example components for Airflow`'.
```

Reply:
Copy and paste error. Fixed!